app/src/main/java/ca/chronofit/chrono/main/circuit/CircuitTimerActivity.kt

```diff
@@ -56,10 +58,12 @@ class CircuitTimerActivity : AppCompatActivity() {
     private var tone: ToneGenerator = ToneGenerator(AudioManager.STREAM_MUSIC, 100)
 
     override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
+        super<AppCompatActivity>.onCreate(savedInstanceState)
         setContentView(R.layout.activity_circuit_timer)
         bind = DataBindingUtil.setContentView(this, R.layout.activity_circuit_timer)
 
+        createNotificationChannel()
```

Comment:
Create all the notifications at the start of the app so Main Activity or even Splash Activity

Reply:
Disabling Timer notification for now. Still creating a notification channel for timer AND stopwatch notification in Main Activity.